util/topsql/reporter/reporter.go

```diff
@@ -252,6 +252,7 @@ func (tsr *RemoteTopSQLReporter) Register(dataSink DataSink) error {
 
 		if len(tsr.dataSinks) > 0 {
 			variable.TopSQLVariable.Enable.Store(true)
```

Comment:
It doesn't need `Enable` anymore if we use `profiler.Consumer`.

But `TopSQLReporter` should export its enabled state to let people know if topsql is enabled.

Reply:
Yes, but we still need this variable for other pkg use.